vpdq/python/tests/test_util.py

```diff
@@ -25,3 +27,49 @@ def read_file_to_hash(input_hash_filename: str) -> t.List[vpdq.VpdqFeature]:
         hash.append(feature)
 
     return hash
+
+
+def generate_hashes(
+    output_folder: Union[Path, str],
+    input_filepaths: List[Union[Path, str]],
+    overwrite: bool = False,
+    downsample_width: int = 0,
+    downsample_height: int = 0,
+) -> None:
+    """
+    Generate vpdq hashes for the test videos
+    and write them to files in output_folder
+
+    If overwrite is set, it will overwrite existing hashes in the folder.
+    """
+    output_folder = Path(output_folder)
+    if not output_folder.exists():
+        print(f"Creating output directory at {output_folder}")
```

Comment:
(ignorable): I usually try to avoid calling print() directly from library functions - `logging` is often preferred if you need chatty libraries, but then you also have to set up logging.

Reply:
Agreed